include/crow/websocket.h

```diff
@@ -157,6 +158,36 @@ namespace crow
                     do_write();
                 });
             }
+            
+            /// Send a binary encoded message in blocking mode.
+            void send_binary_blocking(const std::string& msg, boost::system::error_code& ec) override
```

Comment:
it would make sense to add a `send_text_blocking()`. Also is it necessary to put `ec` there?

Reply:
see my answer above regarding check_destroy()